app/page.tsx

```
@@ -1,11 +1,15 @@
-import { DashedHero } from "@/components/hero/dashed-hero"
-import { Footer } from "@/components/footer"
+import { ExploreCard } from "@/components/explore-card";
+import { MissionCard } from "@/components/mission-card";
+import { Newsletter } from "@/components/newsletter";
+import { FadeIn } from "@/components/animations/fade-in";
+import { AnimatedText } from "@/components/animations/animated-text";
+import { CountUp } from "@/components/animations/count-up";
+import { DashedHero } from "@/components/Hero/dashed-hero";
+import { Footer } from "@/components/footer";
+
+export const sum = (a: any, b: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)